pages/search/views/index.jsx

```diff
@@ -25,7 +25,7 @@ const formatters = {
         return establishments.map((establishment, i) => [
           // separate links with line breaks #hack
```

Comment:
I know this isn't part of this PR but.... 🤢

Reply:
done